app/uk/gov/hmrc/fileupload/controllers/EnvelopeController.scala

```diff
@@ -94,4 +95,11 @@ class EnvelopeController(nextId: () => EnvelopeId,
       case Xor.Left(error) => InternalServerError("It was not possible to retrieve in progress files")
     }
   }
+
+  def nonStubFunctionsWithBasicAuth(returnFunctions: => Future[Result])(request: RequestHeader): Future[Result] = {
```

Comment:
returnFunctions? I don't understand what this variable represents by looking at it